<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">#page{
	border-image: url(:/Image/CL Inspection Monitor 2-5-01.jpg);
}</string>
     </property>
     <widget class="Button_TivanEX" name="btn_backtosetting">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>870</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_backtosetting{
	border-image: url(:/Image/B.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Button_TivanEX" name="btn_power">
      <property name="geometry">
       <rect>
        <x>1113</x>
        <y>868</y>
        <width>78</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_power{
	border-image: url(:/Image/System Control Off.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save_D5">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>620</y>
        <width>77</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_save_D5 {
	border-image: url(:/Image/Report.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset_D5">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>730</y>
        <width>77</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/Image/Asset 6.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>591</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_CL"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>610</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_D5_max"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>610</y>
        <width>131</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Statistic_MonitorLayout_D5"/>
     </widget>
     <widget class="QLabel" name="label_Sample_Name">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Satoshi Variable</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_Sample_Name{

color: rgb(255, 255, 255);
padding:10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>100</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="Camera_Layout_CL"/>
     </widget>
     <widget class="QPushButton" name="btn_save_CL">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>110</y>
        <width>77</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_save_CL{
	border-image: url(:/Image/Report.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset_CL">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>230</y>
        <width>77</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_reset_CL{

border-image: url(:/Image/Asset 6.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>610</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="Camera_Layout_D5"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>100</y>
        <width>131</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Statistic_MonitorLayout_CL"/>
     </widget>
     <widget class="QLabel" name="label_CL_Accept_num">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>220</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_CL_Accept_num{
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_CL_Accept_color">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>230</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_CL_Accept_color{
	
	background-color: rgb(0, 234, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_CL_Reject_color">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>280</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_CL_Reject_color{
	
	background-color: rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_CL_Reject_num">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>270</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_CL_Reject_num{
	
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_D5_Accept_color">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>730</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D5_Accept_color{
	
	background-color: rgb(0, 234, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_D5_Reject_color">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>780</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D5_Reject_color{
	background-color: rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_D5_Accept_num">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>720</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D5_Accept_num{
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_D5_Reject_num">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>770</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D5_Reject_num{
	
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{

background-color : rgba(255,255,255,0);

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>360</y>
        <width>131</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Statistic_MonitorLayout_D4"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>360</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="Camera_Layout_D4"/>
     </widget>
     <widget class="QPushButton" name="btn_Report_D4">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>360</y>
        <width>77</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_Report_D4{
	border-image: url(:/Image/Report.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset_D4">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>480</y>
        <width>77</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_reset_D4{
	border-image: url(:/Image/Asset 6.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_D4_max"/>
     </widget>
     <widget class="QLabel" name="label_D4_Accept_num">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>460</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D4_Accept_num{
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_D4_Reject_num">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>510</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D4_Reject_num{
	
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_D4_Accept_color">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>470</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D4_Accept_color{
	
	background-color: rgb(0, 234, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_D4_Reject_color">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>520</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_D4_Reject_color{
	background-color: rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_CL_Reject">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>153</y>
        <width>72</width>
        <height>118</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_CL_Reject
{
	border-image: url(:/Image/CL-R-OFF.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_FL1_Reject">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>403</y>
        <width>72</width>
        <height>118</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_FL1_Reject
{
	border-image: url(:/Image/FL1-R-OFF.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_FL2_Reject">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>653</y>
        <width>72</width>
        <height>118</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_FL2_Reject
{
	border-image: url(:/Image/FL2-R-OFF.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>460</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_D4_min"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>710</y>
        <width>571</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_D5_min"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button_TivanEX</class>
   <extends>QPushButton</extends>
   <header>Button_TivanEX.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
